client/services/placeApi.js

```diff
@@ -7,10 +7,29 @@ axios.defaults.headers.common = {
   'Content-Type': 'application/json',
 };
 
+const parseParams = (params) => {
```

Comment:
this is a very typical specific example for generic usage... how do you know for a fact that for object, we always wanted to pass them over as a 'bla, bla' format? a simple mapper will be much more clear, readable and testable. For code that if a normal dev looks at it and cant figure out what its doing within a few seconds that's worth considering refactoring